ecs-deploy

```diff
@@ -135,6 +137,87 @@ function deployService() {
   printf "Done deploying service %s to %s...\n" $SERVICE $CLUSTER
 }
 
+waitForSSMCommand() {
+  every=2
+  count=0
+
+  while [ $count -lt $TIMEOUT ]; do
+    details=$($SSM_CLI list-command-invocations --command-id "$SSM_COMMAND_ID" --details)
+    COMMAND_OUTPUT=$(echo "$details" | jq -r '.CommandInvocations[].CommandPlugins[].Output')
+    COMMAND_EXIT_CODE=$(echo "$details" | jq -r '.CommandInvocations[].CommandPlugins[].ResponseCode')
+    if [ $COMMAND_EXIT_CODE -gt -1 ]; then
+      break
+      sleep $every
```

Comment:
Isn't this dead code? I never used `break` in bash, but sleep seems to be never executed?

Reply:
Ah yes, I just accidentally switched the two lines – this should come after the `else`, see https://github.com/formunauts/drone-ecs-deploy-advanced/pull/1/commits/7628e8f4e1f01ca10787394f243adba4e8567710. Then it hopefully makes more sense :wink: